RELEASE/scripts/autoscend/iotms/mr2018.ash

```diff
@@ -569,6 +569,10 @@ boolean cheeseWarMachine(int stats, int it, int eff, int potion)
 	{
 		return false;
 	}
+	if(!auto_is_valid($item[Bastille Battalion Control Rig]))
```

Comment:
I would change the call to `is_unrestricted()` at line 564 to this instead (GitHub doesn't let me suggest for that line).